crates/wash-lib/src/cli/logs.rs

```diff
@@ -0,0 +1,103 @@
+use anyhow::{bail, Result};
+use clap::Parser;
+
+use crate::{
+    cli::{CliConnectionOpts, CommandOutput},
+    common::{boxed_err_to_anyhow, find_host_id},
+    config::WashConnectionOptions,
+};
+
+#[derive(Clone, Debug, Parser)]
+pub struct GetLoggingConfigCommand {
+    #[clap(flatten)]
+    pub opts: CliConnectionOpts,
+
+    /// ID of host to update the actor on. If a non-ID is provided, the host will be selected based
+    /// on matching the prefix of the ID or the friendly name and will return an error if more than
+    /// one host matches.
+    #[clap(name = "host-id")]
+    pub host_id: String,
+}
```

Comment:
When I read `wash logs get snowy`, as a user I think "I want to retrieve the logs for a specific host", not "I want to retrieve the log level for a specific host". Given the fact that `set` sets the log level and it doesn't make sense to "set" the logs for a host, I could reason about this, but did you consider any alternatives for the top level command that may disambiguate this?

My biggest issue is just that users will likely look at `wash logs` and think they can retrieve logs, but find out that they can't after inspecting it.

Reply:
I did think about this, and don't love where I ended up. You're right that `wash logs get` reads more like "get the logs for this host", which might be a useful feature, but is not what this is intended for.

I'm trying to avoid breaking changes shortly after 1.0. Today, the only use case this new feature enables is overriding the log level. But there might be other functionality we want, e.g. reconfiguring the OTEL endpoint, or updating sampling configuration, etc. We don't have any of that functionality today, but I wanted to make the command (and underlying payload) generic enough that we could simply add options to it, rather than scrapping it in favor of an entirely new command/subject/payload/etc.

In that light, can you help me think of a better name? Maybe `wash otel [get|set]`? I suppose since this is an advanced debug feature that won't be used all that often, we could even afford to be verbose: `wash otel-config get`? (`otel` is motivated by my journey into OTEL logging land today). E.g.:
- `wash otel-config get --host-id foo log-level`
- `wash otel-config set --host-id bar trace-sampling 0.1`
- `wash otel-config set --host-id baz log-format structured`

Reply:
This honestly sounds like a broader story around setting host configuration at runtime, like we've been setting up labels and log levels and perhaps we need a more general purpose set of config vars... I will noodle on this a bit but I don't have a great answer right now. Maybe the command we could use is "wash log level set|get"? That is intuitive but doesn't provide extensibility